modules/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -19,21 +19,24 @@ spec:
   domain:
     devices:
       interfaces:
-      - masquerade: {} <1>
-        name: default
-      - name: <sriov-network> <2>
+      - name: default <1>
+        masquerade: {} <2>
+      - name: nic1 <3>
         sriov: {}
   networks:
-  - name: default <3>
+  - name: default <4>
     pod: {}
-  - multus:
-    name: <sriov-network> <4>
+  - name: nic1 <5>
+    multus:
+        networkName: <sriov-network> <4>
 ...
 ----
-<1> The `masquerade` binding to the default Pod network.
-<2> The name of the SR-IOV network.
-<3> The default Pod network.
-<4> The name of the SR-IOV network. This is the same value as the `interfaces.name`.
+<1> A name for the interface that is connected to the Pod network.
+<2> The `masquerade` binding to the default Pod network.
+<3> A name for the SR-IOV interface.
+<4> The name of the interface used in <1>.
+<5> The name of the SR-IOV interface. This must be the same as <3>.
```

Comment:
Unfortunately the second callouts here don't build. 
```suggestion
<4> The name of the Pod network interface. This must be the same as the `interfaces.name` that you defined earlier.
<5> The name of the SR-IOV interface. This must be the same as the `interfaces.name` that you defined earlier..
```

Reply:
@dankenigsberg - Did you want to take this suggestion or rewrite to avoid use of the second `<1>`/`<3>` in the callouts ?
That's the only thing preventing merge atm

Reply:
I missed your former comment. Can we not refer to the precise number? We have two `intrefaces.name` and we need to differenciate between them.

Reply:
but I cannot find a proper text right now, Ill take yours!